bzt/engine.py

```diff
@@ -1354,6 +1355,9 @@ def get_data_sources(self):
 
             yield source
 
+    def get_keystore_config(self):
```

Comment:
This is JMeter-specific thing, it can't be in the core

Reply:
Just move it into JMeter part, there's nothig bad in doing `scenario.get(self.FIELD_KEYSTORE_CONFIG)` there. And `FIELD_KEYSTORE_CONFIG` is name of JMeter-specific option, also belongs to JMeter module.

Reply:
Makes sense. Change done and committed.. Thank you!